src/fidelio.c

```diff
@@ -40,19 +41,23 @@ void system_boot(void)
     set_sys_clock_48mhz();
     
     /* Setting GPIOs for Led + Button */
-    gpio_init(U2F_LED);
-    gpio_set_dir(U2F_LED, GPIO_OUT);
+    int rc = u2f_led_init();
```

Comment:
Don't mix declarations and code: please declare "int rc" at the top of the block.

Reply:
Sorry - copied from elsewhere (lazy)